news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/trigger.jpg)](https://osu.ppy.sh/s/555076)
+
+I have to say it. This map is definetly the most fun experience I had in a while from the ranked pool and I can only say thanks and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The SVs on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple LN patterns this map just has everything you want to be the map of the month... and its art!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/re-lm.jpg)](https://osu.ppy.sh/s/588066)
+
+This map caught my eye for a few reasons. The most obvious being the heavy use of SV, which is thoughtfully crafted to be intuitive and smooth, which is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. Though the SV is not the main reason I think this map is particularly deserving of a spotlight. Lude doesn't shy away from LN usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity and originality, which is what I personally love to see. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves, makes this a suitable pick for May's spotlight.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/black-or-white.jpg)](https://osu.ppy.sh/s/565779)
+
+Let me introduce you another extreme 7K marathon map by Kawawa. I really like how the map structure flows nicely with the song intensity. From the map itself, i can see the mapper have a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun. The mapper really did a good job on hightlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shine.jpg)](https://osu.ppy.sh/s/456289)
+
+For his 5th ranked map, Frim4503 offers us another 5K marathon, this time for a drum & bass love live opening remix !
+As usual we have a mix of LN and chords playing really good, making it a really fun map to try big accuracy scores. SV are really lightly used aswell with a quite unique style, focussing on the voice before each kiai. Without changing much of music/mapping style, Frim4503 show us one more time that anti-meta is not dead with a really accessible and catchy map.
```

Comment:
Mapper names need to be profile linked, etc. Show should be "shows", focussing should be "focus". Run the thing through a spell checker set to EN-GB.